node-api/backend/spec.go

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: BUSL-1.1
+//
+// Copyright (C) 2025, Berachain Foundation. All rights reserved.
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file of this repository and at www.mariadb.com/bsl11.
+//
+// ANY USE OF THE LICENSED WORK IN VIOLATION OF THIS LICENSE WILL AUTOMATICALLY
+// TERMINATE YOUR RIGHTS UNDER THIS LICENSE FOR THE CURRENT AND ALL OTHER
+// VERSIONS OF THE LICENSED WORK.
+//
+// THIS LICENSE DOES NOT GRANT YOU ANY RIGHT IN ANY TRADEMARK OR LOGO OF
+// LICENSOR OR ITS AFFILIATES (PROVIDED THAT YOU MAY USE A TRADEMARK OR LOGO OF
+// LICENSOR AS EXPRESSLY REQUIRED BY THIS LICENSE).
+//
+// TO THE EXTENT PERMITTED BY APPLICABLE LAW, THE LICENSED WORK IS PROVIDED ON
+// AN “AS IS” BASIS. LICENSOR HEREBY DISCLAIMS ALL WARRANTIES AND CONDITIONS,
+// EXPRESS OR IMPLIED, INCLUDING (WITHOUT LIMITATION) WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, NON-INFRINGEMENT, AND
+// TITLE.
+
+package backend
+
+import (
+	"github.com/berachain/beacon-kit/errors"
+	"github.com/berachain/beacon-kit/node-api/handlers/config/types"
+)
+
+func (b *Backend) Spec() (*types.SpecData, error) {
+	chainSpec := b.ChainSpec()
+	if chainSpec == nil {
+		return nil, errors.New("chain spec not found")
+	}
+	return &types.SpecData{
+		DepositContractAddress: chainSpec.DepositContractAddress(),
+		// Network ID is same as eth1 chain ID.
+		DepositNetworkID:                chainSpec.DepositEth1ChainID(),
```

Comment:
Using Eth1 chain ID. The other approach is to introduce DepositNetworkID in the chain spec, and have its value equal to Chain ID.
As the current approach will reduce LoC as it has the same value anyway. I am open to suggestions.